src/hardware/ad7172.rs

```diff
@@ -67,6 +62,15 @@ pub mod Ifmode {
     }
 }
 
+/// ADC GPIOCON register settings.
+#[allow(non_snake_case)]
+pub mod Gpiocon {
+    pub mod SyncEn {
+        pub const ENABLED: u32 = 1 << 11; // enable sync input.
```

Comment:
General remark: I would be more deliberate with the comments and choose where to use them and what to write carefully. If they just duplicate what the code itself already says, then they not just redundant but worse: they will (as you have seen before) synchronization.